### ISSUE TITLE
Add test to CI

### ISSUE THREAD

**Issue by jason-fox** [ramp-eu/AGILPLAS#4]
Configure a CI system of your choice to re-add the missing badge
Delete the `.travis` if not using Travis CI. Consider adding a GitHub Action like the [Template Project](https://github.com/ramp-eu/TTE.project1/blob/master/.github/workflows/ci.yml) - this could also run the code analysis which is one of the missing CII points.

Your CI badge would then look like this:

```markdown
[![CI](https://github.com/ramp-eu/JS2SF/workflows/CI/badge.svg)](https://github.com/ramp-eu/JS2SF/actions?query=workflow%3ACI)
```

The `test` folder currently holds the original dummy Mocha test - please delete and add a test framework of you choice

**Comment by jaclavijo** [comment]
Code analysis is set up with Codacy. An extended CI may be implemented in the future.

**Comment by jaclavijo** [comment]
Automated Docker image building and pushing CI has been implemented 67cf83bfba884b05dd943f30feb042e9e1c81882
